### PR TITLE
Sort Orders as they are added

### PR DESCRIPTION
**Pull request by Jaspertzx**
The orders are not sorted chronologically.

This may confuse users or make the application hard for them to check for deadline.

Add sort command whenever an Order is added.

This will ensure the entire Order list stays sorted.

### CODE REVIEW

**Comment by Jaspertzx** [comment]
Resolves #105

**Review by kailinteoo — APPROVED** [review]
LGTM! thanks for doing the sort feature for order!!

**Review by rertyy — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -76,6 +76,18 @@ public Set<Order> getOrders() {
...
+     * Returns a new Person object, with the specified Order added to the orders object.
+     *
+     * @param order the order to be added
+     * @return new Person object
+     */
+    public Person addOrder(Order order) {
+        Set<Order> newOrders = new HashSet<>(orders);
+        newOrders.add(order);
+        return new Person(this.name, this.phone, this.email, this.address, this.getTags(), newOrders);
+    }
```
Comment (rertyy) [inline src/main/java/seedu/address/model/person/Person.java:89]: This is good yes

**Pull request merged**